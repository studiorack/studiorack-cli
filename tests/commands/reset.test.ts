import { expect, test } from 'vitest';
import { cli } from '../shared';
import { RegistryType } from '@open-audio-stack/core';

test('Reset packages', async () => {
  expect(cli(RegistryType.Plugins, 'reset')).toMatchSnapshot();
});
