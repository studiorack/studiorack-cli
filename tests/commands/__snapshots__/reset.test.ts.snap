// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Reset packages 1`] = `"undefined"`;
